client/src/componenets/AllCharacters.js

```diff
@@ -1,8 +1,40 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+import Character from './Character';
+import CharacterModel from '../models/CharacterModel'
+import Styles from './AllCharacters.module.css'
+
+import useAxios from '../hooks/useAxios';
+
+const AllCharacters = (props) => {
+  const { pageNum } = props;
+  const { fetchData } = useAxios();
+  const [charctersData, setCharctersData] = useState([]);
+
+  useEffect(async () => {
+    await fetchData(`allCharacters/${pageNum}`).then(res => {
+      setCharctersData(res.data.characters);
+    });
+  }, [pageNum]);
+
+  const renderCharacters = () => {
+    return charctersData.map(item => <Character
```

Comment:
Please destructuring the item to 
`{ name, id, status, species, origin, image }`